opencensus/trace/ext/django/middleware.py

```diff
@@ -216,3 +224,13 @@ def process_response(self, request, response):
             log.error('Failed to trace request', exc_info=True)
         finally:
             return response
+
+    def _get_service_name(self, params):
+        _service_name = params.get(
+            SERVICE_NAME, None)
+
+        if _service_name is None:
+            _service_name = params.get(
+                ZIPKIN_EXPORTER_SERVICE_NAME, 'my_service')
```

Comment:
If we want to send Zipkin spans to OC-Agent, they need to be converted to OC span proto first. Is it the case in Django?

If that's not the case, please do not add Zipkin as a service endpoint for now. We plan to add an interceptor to accept Zipkin spans in OC-Agent.

Reply:
this is not about sending the spans, it's just a django/flask app setting for the service name on the exporter.
There was a setting ZIPKIN_EXPORTER_SERVICE_NAME, but I want to generalize it and rename to SERVICE_NAME

Reply:
Got it, thanks for the explaination.